### PR TITLE
How gitlab tests

### PR DESCRIPTION
**Pull request by SachinJoshiTest**
Sharing the testing approach of a wonderful all-remote company which has never missed a release date.

### CODE REVIEW

**Comment by abhivaikar** [comment]
Hi @SachinJoshiTest , GitLab is already part of the list and there are many resources from GitLab added in there. Can you please check? If your list has any new resources please feel free to update the existing list.

**Comment by SachinJoshiTest** [comment]
> Hi @SachinJoshiTest , GitLab is already part of the list and there are many resources from GitLab added in there. Can you please check? If your PR has any new resources please feel free to update the existing list.

Sure i will update extra references in that link

**Comment by SachinJoshiTest** [comment]
> > Hi @SachinJoshiTest , GitLab is already part of the list and there are many resources from GitLab added in there. Can you please check? If your PR has any new resources please feel free to update the existing list.
> 
> Sure i will update extra references in that link

Updated the current GitLab reference

File: README.md [hunk 1]
```diff
@@ -339,13 +339,27 @@ Happy Learning!
...
+* [End-to-End testing of GitLab](https://docs.gitlab.com/ee/development/testing_guide/end_to_end/)
+* [Different testing levels](https://docs.gitlab.com/ee/development/testing_guide/testing_levels.html)
+* [Test Engineering and planning](https://about.gitlab.com/handbook/engineering/quality/test-engineering/)
+* [Performance Testing Gitlab](https://about.gitlab.com/blog/2020/02/18/how-were-building-up-performance-testing-of-gitlab/)
+* [Testing best practices](https://docs.gitlab.com/ee/development/testing_guide/best_practices.html)
+* [How Gitlab is able to achieve everthing is with it's Engineering workflow](https://about.gitlab.com/handbook/engineering/workflow/)
+* [Quick overview of GitLab](https://www.enovate.co.uk/blog/2015/11/25/gitlab-review)
+
+#### Videos
+* [GitHub v/s GitLab](https://www.youtube.com/watch?v=s8DCpG1PeaU)
```
Comment (abhivaikar) [inline README.md:361]: Not needed as it does not talk about testing at Gitlab.

File: README.md [hunk 2]
```diff
@@ -339,13 +339,27 @@ Happy Learning!
...
+* [Devops best practices with GitLab](https://www.youtube.com/watch?v=6FvcPLlJapI)
```
Comment (abhivaikar) [inline README.md:362]: Same here.

File: README.md [hunk 3]
```diff
@@ -339,13 +339,27 @@ Happy Learning!
...
 * [How our QA team leverages GitLab’s performance testing tool (and you can too)](https://about.gitlab.com/blog/2020/02/18/how-were-building-up-performance-testing-of-gitlab/)
 * [How GitLab switched to Headless Chrome for testing](https://about.gitlab.com/blog/2017/12/19/moving-to-headless-chrome/)
+* [End-to-End testing of GitLab](https://docs.gitlab.com/ee/development/testing_guide/end_to_end/)
+* [Different testing levels](https://docs.gitlab.com/ee/development/testing_guide/testing_levels.html)
+* [Test Engineering and planning](https://about.gitlab.com/handbook/engineering/quality/test-engineering/)
+* [Performance Testing Gitlab](https://about.gitlab.com/blog/2020/02/18/how-were-building-up-performance-testing-of-gitlab/)
+* [Testing best practices](https://docs.gitlab.com/ee/development/testing_guide/best_practices.html)
+* [How Gitlab is able to achieve everthing is with it's Engineering workflow](https://about.gitlab.com/handbook/engineering/workflow/)
+* [Quick overview of GitLab](https://www.enovate.co.uk/blog/2015/11/25/gitlab-review)
```
Comment (abhivaikar) [inline README.md:358]: This one too. It does not talk about testing at Gitlab.

**Pull request closed without merging**